src/components/Article/styled.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+import { BLUE, MarginContainer, P, H2, H3 } from "../../styles/styles";
```

Comment:
can you remove these unused imports? you should only need to import the `P` here